db/post_migrate/20240711102255_ensure_consistency.rb

```diff
@@ -27,7 +27,9 @@ def up
     # delete votes associated with no user
     DB.exec(<<~SQL)
       DELETE FROM topic_voting_votes
-      WHERE user_id NOT IN (SELECT id FROM users)
+      USING users
+      WHERE topic_voting_votes.user_id = users.id
+      AND users.id IS NULL;
```

Comment:
This condition seems impossible to me. `users.id` can never be null... so how would this ever match anything? 🤔